src/containers/Entry/AddEntry/index.js

```diff
@@ -0,0 +1,171 @@
+    </Row>
+
+    <Form form={form} layout="vertical" onFinish={handleSubmit}>
+      <Row gutter={20}>
+        <Col span={16}>
+          <Form.Item
+            label="Produto"
+            name="productId"
+            rules={[{ required: true }]}
+          >
+            <Select
+              onChange={(_, { props: { serial } }) => onChange(serial)}
+              onSearch={onSearchProduct}
+              optionFilterProp="children"
+              placeholder="Selecione o produto"
+              showSearch
+            >
+              {map(
+                renderOption({
+                  children: 'name',
+                  key: 'id',
+                  value: 'id',
+                }),
+                productList
+              )}
+            </Select>
+          </Form.Item>
+        </Col>
+        <Col span={8}>
+          <Form.Item
+            label="Estoque"
+            name="stockBase"
+            rules={[{ required: true }]}
+          >
+            <Select placeholder="Selecione a base de estoque">
+              {map(
+                renderOption({
+                  children: 'name',
+                  key: 'name',
+                  value: 'name ',
+                }),
+                baseList
+              )}
+            </Select>
+          </Form.Item>
+        </Col>
+      </Row>
+
+      <Row gutter={20}>
+        <Col span={16}>
+          <Form.Item
+            label="Fornecedor"
+            name="companyId"
+            rules={[{ required: true }]}
+          >
+            <Select
+              optionFilterProp="children"
+              onSearch={onSearchProvider}
+              placeholder="Selecione o Fornecedor"
+              showSearch
+            >
+              {map(
+                renderOption({
+                  children: 'razaoSocial',
+                  key: 'id',
+                  value: 'id',
+                }),
+                providerList
+              )}
+            </Select>
+          </Form.Item>
+        </Col>
+        <Col span={4}>
+          <Form.Item
+            label="Quantidade"
+            name="amountAdded"
+            rules={[{ required: true }]}
+          >
+            <InputNumber min={1} placeholder="-" />
+          </Form.Item>
+        </Col>
+        <Col span={4}>
+          <Form.Item
+            label="Aguard. análise"
+            name="analysis"
+            valuePropName="checked"
+          >
+            <Switch />
+          </Form.Item>
+        </Col>
+      </Row>
+
+      <Row>
+        <Col span={24}>
+          <Form.Item
+            noStyle
+            shouldUpdate={(prevValues, currentValues) =>
+              prevValues.analysis !== currentValues.analysis ||
+              prevValues.productId !== currentValues.productId
+            }
+          >
+            {({ getFieldValue }) => {
+              return (
+                !getFieldValue('analysis') &&
+                alllowInsertSerilNumber && (
```

Comment:
```suggestion
                allowInsertSerialNumber && (
```

mudar tudo que esteja "alllowSerilNumber" para "allowSerialNumber"